lib/forwarder.dart

```diff
@@ -57,6 +61,10 @@ class WipForwarder {
       Map<String, dynamic> params = json['params'];
       bool processed = false;
 
+      if (method.contains('reakpoint')) {
```

Comment:
Yes. The 'b' is capitalized in the commands I looked at (Debugger.setBreakpoint, etc), but I didn't feel like looking through all of the commands to figure out if it is always capitalized, so I just dropped it.